kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,12 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangowhiteNoise):
+
+    def __init__(self, app):
+        DjangoWhiteNoise.__init__(self, app)
+
+    # Correct some MIME types
+    EXTRA_MIMETYPES = (
+            ('application/font-woff', '.woff'),
+            ('application/font-woff2', '.woff2'),)
```

Comment:
In terms of adding to the existing tuple, rather than replacing it? I started down that path, but then realized that the default `woff` MIME type that WhiteNoise uses is wrong. I'll have to open an issue with them about it.

https://github.com/evansd/whitenoise/blob/b3974c2ea80e93ba086331cff447a09a427a1a9f/whitenoise/base.py#L72-L73